language/e2e-tests/src/tests/validator_set_management.rs

```diff
@@ -181,8 +187,10 @@ fn validator_set_operator_set_key_reconfigure() {
         libra_root_account
             .transaction()
             .script(encode_create_validator_account_script(
+                0,
```

Comment:
This transaction with the previous one has a zero nonce? Is it allowed, is not failing as a "replay"?

Reply:
So 0 is actually a special nonce which is always accepted :) (look at SlidingNonce module) - so we can just 0 for tests.